specification/2.0/Specification.adoc

```diff
@@ -818,13 +818,13 @@ Buffers and buffer views do not contain type information. They simply define the
 [[glb-stored-buffer]]
 ==== GLB-stored Buffer
 
-glTF asset **MAY** use GLB file container to pack glTF JSON and one glTF buffer into one file. Data for such buffer is provided via GLB-stored `BIN` chunk.
+The glTF asset **MAY** use a GLB file container to pack glTF JSON and one glTF buffer into one file. Data for such a buffer is provided via the GLB-stored `BIN` chunk.
```

Comment:
Are you sure about "use a GLB file container" here? A similar sentence on L474 says "use the GLB container".

Reply:
Agreed.  I changed it to "use the GLB container."